src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1040,49 +1038,73 @@ private enum AssignmentKind
             bool useLegacyWarnings,
             AssignmentKind assignmentKind = AssignmentKind.Assignment,
             ParameterSymbol parameterOpt = null,
-            Conversion conversion = default,
             Location location = null)
         {
+            // Callers should apply any conversions before calling this method
```

Comment:
Should this be in the doc comment instead?

Reply:
Added.

---
In reply to: [347636495](https://github.com/dotnet/roslyn/pull/39778#discussion_r347636495) [](ancestors = 347636495)